src/Tribe/Aggregator/API/Import.php

```diff
@@ -107,9 +107,19 @@ public function get( $import_id, $data = array() ) {
 	 * @return stdClass|WP_Error
 	 */
 	public function create( $args ) {
-		$response = $this->service->post_import( $args );
+		return $this->service->post_import( $args );
+	}
 
-		return $response;
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$import_id" missing
* :no_entry_sign: Doc comment for parameter "$args" missing

<sub>hash: 7bdf5347d10666c43136740c0818a700</sub>